widgets/button/styles/atom.less

```diff
@@ -55,6 +58,13 @@
 			.gradient(lighten(@button_color, 2%), lighten(darken(@button_color, 10%), 2%), lighten(@button_color, 2%));
 			border-color: lighten(lighten(@border_color, 2%), 2%) lighten(@border_color, 2%) darken(lighten(@border_color, 2%), 3%) lighten(@border_color, 2%);
 			color: lighten(@text_color, 2%);
+			
+			& when ( @hover_background_color != default ) or ( @hover_text_color != default ) {
```

Comment:
Unfortunately, `or` isn't valid. Less uses a comma instead of `or`. My example was, in hindsight, misleading. [Here's a better example ](https://github.com/siteorigin/so-widgets-bundle/blob/986ae26a964f9363023613162330dc87802eeac4/widgets/tabs/styles/default.less#L48-L66 )and more information on this can be read [here](http://lesscss.org/features/#mixins-feature-guarded-namespaces).

To view LESS changes you'll need to clear the SiteOrigin Widgets Bundle CSS Cache. This can be done by navigating to **wp-content/uploads/siteorigin-widgets** and delete all of the CSS files in that directory. If the widget doesn't display correctly after doing this, there's a LESS syntax error.

Reply:
`!=` also isn't valid in LESS. You need to use NOT instead. Yeah, I don't know why `OR` is valid but `!=` isn't either.
For more information please view http://lesscss.org/features/#mixins-feature-guarded-namespaces and scroll down to the Guard Logical Operators heading..